lib/remote/apilistener.cpp

```diff
@@ -486,13 +491,40 @@ bool ApiListener::AddListener(const String& node, const String& service)
 	Log(LogInformation, "ApiListener")
 		<< "Started new listener on '[" << localEndpoint.address() << "]:" << localEndpoint.port() << "'";
 
-	IoEngine::SpawnCoroutine(io, [this, acceptor](asio::yield_context yc) { ListenerCoroutineProc(yc, acceptor); });
+	/* The Asio strand for the listener socket.
+	 */
```

Comment:
```suggestion
	// The Asio strand for the listener socket.
```

Or in case you only added this comment because I asked for one documenting the scope of the strand earlier: When I asked for that, the strand was a class member with it's use spread across more functions. Now it's all done here in this function in close proximity and also the Asio shutdown signal that was additionally guarded is gone, so I'd also be fine with removing the comment.